node_locations:
  pos_1: 
    - 0.8787446022033691
    - 3.0359625816345215
    - -0.13255167007446292
  pos_2: 
    - 1.458296537399292
    - 1.2651097774505615
    - -0.37433350086212164

node_neighbors: 
  pos_1:
    - pos_2
  pos_2:
    - pos_1